integration-tests/example.env

```diff
@@ -3,8 +3,8 @@
 
 # Test Settings
 export KEEP_ENVIRONMENTS="Never" # Always | OnFail | Never
-export CHAINLINK_IMAGE="public.ecr.aws/chainlink/chainlink"
-export CHAINLINK_VERSION="1.5.1-root"
+export CHAINLINK_IMAGE="795953128386.dkr.ecr.us-west-2.amazonaws.com/chainlink"
```

Comment:
Is this publicly accessible or only internal? I ask because I just went on a mission to remove our aws account number from a bunch of places and now we are adding it back in here.

Reply:
Reverted it - https://github.com/smartcontractkit/chainlink/pull/7088